lib/navigation/view.ts

```diff
@@ -19,8 +19,7 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-/* eslint-disable no-use-before-define */
-import type { Folder, Node } from '@nextcloud/files'
+import type { Folder, Node } from '../'
```

Comment:
Maybe use the full path instead?
```suggestion
import type { Folder } from '../files/folder'
import type { Node } from '../files/node
```